### ISSUE TITLE
Consider linking Visual C++ Redistributable here and on the main website

### ISSUE THREAD

**Issue by PixelEater** [picotorrent/picotorrent#551]
Given that on a fresh install of Windows 10 you are greeted immediately with messages PicoTorrent cannot run for lack of MSVCP140.dll and VCRUNTIME140.dll, it maybe friendly to link https://www.microsoft.com/en-us/download/details.aspx?id=48145 or similar from the README.md and front page and download page of picotorrent.org.

**Comment by vktr** [comment]
Is this for the portable zip or the installer? The installer should bundle
the redistributable.

But it’s a good suggestion to add a note anyways. I’ll make sure to do
that.

**Comment by PixelEater** [comment]
Oh I feel a bit silly, shoulda noticed that. Now, I wanted to test that on my machine and I did run into an unusual problem which would cause the installer to fail. Here's the order that happened in.

I had been playing with the portable PicoTorrent.exe, but also had installed Visual Studio 2015 (before I noticed 2017) and I had installed PicoTorrent x64 proper to Program Files. I uninstalled it and I also uninstalled four Visual Studio packages.

Then I tried installing PicoTorrent stable and nightly, and both installers hung on the VC install then errored to 0x80091007 - The hash value is not correct. I thought a reboot might help, but it did not. Now that I manually reinstalled the 2017 VC redistributable, I was able to complete the installer. I'm really not sure if this situation was strictly created by uninstalling Visual Studio through Windows' own Apps & features menu or what.
